Deadreyo/SemiColon-backend
# Add user endpoint to the API

## Description

create user endpoint in the API so that it can be used for CRUD operations on the users

## Testing Instructions (Optional)
+ before running the tests make sure to have .env file having env variable called `TEST_DB_URL=''` and add your MongoDB URL between the '' 
+ to run the provided tests use `npm run test`
